core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.scss

```diff
@@ -3,14 +3,19 @@
 :host {
     display: block;
 
+    &.hide {
+        opacity: 0;
+    }
+
     ::ng-deep {
```

Comment:
We need to discuss if these styles should be on global. 

cc: @fmontes

Reply:
Since "raised" button in not in our design system I think we can keep it here.